doc/api/fs.md

```diff
@@ -2529,6 +2529,18 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the `oldPath` of a file and rename it to the
+pathname provided as `newPath`.
```

Comment:
How about making something to this effect the first sentence of the documentation, rather than making the user have to read to the second paragraph? Something like the following would sound good to me, and be consistent with the rest of the file (e.g. [`chmod()`](https://github.com/nodejs/node/blob/master/doc/api/fs.md#fschmodpath-mode-callback)).

```markdown
Take the `oldPath` of a file and rename it to the pathname provided as
`newPath` asynchronously. No arguments other than a possible exception are
given to the completion callback.

See also: rename(2).
```

Also, the documentation could be more helpful if it specifies what happens if `newPath` already exists as a file, or directory. Is the callback function called with an exception, or is the original file overwritten?

Reply:
Further edit to @TimothyGu's suggestion to make the first sentence more concise:

```markdown
Asynchronously rename file at `oldPath` to the pathname provided as
`newPath`. No arguments other than a possible exception are
given to the completion callback.

See also: rename(2).
```